### PR TITLE
Authorization and Stats API

### PR DESCRIPTION
**Pull request by dihan0604**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@dihan0604__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (dihan). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by dihan0604** [comment]
Check is failling due to package is not published. I am waiting PR https://github.com/Azure/azure-sdk-for-net/pull/1614 to be merged to SDK to generate package so that check for powershell can pass.

**Comment by jianghaolu** [comment]
Left one comment on the PR Azure/azure-sdk-for-net#1614. Please address that and we will merge it.

Once that package is published, we can re-run the test here. Please do so asap if you want this pull request to be included in 1.1.0 release.

File: src/ServiceManagement/ExpressRoute/Commands.ExpressRoute/ExpressRouteClient.cs [hunk 1]
```diff
@@ -55,6 +55,37 @@ public ExpressRouteClient(ExpressRouteManagementClient client)
...
+            return new AzureDedicatedCircuitStats()
+            {
+                PrimaryBytesIn = stats.PrimaryBytesIn,
+                PrimaryBytesOut = stats.PrimaryBytesOut,
+                SecondaryBytesIn = stats.SecondaryBytesIn,
+                SecondaryBytesOut = stats.SecondaryBytesOut
+            };
+        }
+
+        public AzureDedicatedCircuitStats GetAzureDedicatedCircuitStatspInfo(Guid serviceKey)
```
Comment (nupurkot) [inline src/ServiceManagement/ExpressRoute/Commands.ExpressRoute/ExpressRouteClient.cs]: typo in function name

**Comment by nupurkot** [comment]
LGTM apart from typo

File: src/ServiceManagement/Services/Commands.Test/Commands.Test.csproj [hunk 2]
```diff
@@ -135,7 +135,7 @@
       <HintPath>..\..\..\packages\Microsoft.WindowsAzure.Management.Compute.12.3.1\lib\net40\Microsoft.WindowsAzure.Management.Compute.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.WindowsAzure.Management.ExpressRoute, Version=0.9.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.WindowsAzure.Management.ExpressRoute.0.18.4-preview\lib\net40\Microsoft.WindowsAzure.Management.ExpressRoute.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.WindowsAzure.Management.ExpressRoute.0.18.6-preview\lib\net40\Microsoft.WindowsAzure.Management.ExpressRoute.dll</HintPath>
```
Comment (jianghaolu) [inline src/ServiceManagement/Services/Commands.Test/Commands.Test.csproj]: 0.18.12?

File: src/ServiceManagement/Services/Commands.Test/packages.config [hunk 3]
```diff
@@ -21,7 +21,7 @@
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="2.0.3" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.Management" version="4.1.1" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.Management.Compute" version="12.3.1" targetFramework="net45" />
-  <package id="Microsoft.WindowsAzure.Management.ExpressRoute" version="0.18.4-preview" targetFramework="net45" />
+  <package id="Microsoft.WindowsAzure.Management.ExpressRoute" version="0.18.6-preview" targetFramework="net45" />
```
Comment (jianghaolu) [inline src/ServiceManagement/Services/Commands.Test/packages.config]: same here?

**Pull request merged**